ModelBundle/DataFixtures/MongoDB/LoadRootNodeDemoData.php

```diff
@@ -0,0 +1,161 @@
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraProductionFixturesInterface;
+use OpenOrchestra\ModelBundle\Document\Area;
+
+/**
+ * Class LoadRootNodeDemoData
+ */
+class LoadRootNodeDemoData extends AbstractFixture implements OrderedFixtureInterface, OrchestraProductionFixturesInterface
+{
+    /**
+     * Load data fixtures with the passed EntityManager
+     *
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager)
+    {
+        $references = array();
+        $references["status-published"] = $this->getReference('status-published');
+        $references["status-draft"] = $this->getReference('status-draft');
+        $references["siteId"] = $this->getReference("site2")->getSiteId();
+        $references["templateId"] = $this->getReference("homepage-template")->getTemplateId();
+        $references["template-areas"] = $this->getReference("homepage-template")->getAreas();
+
+        $routePattern = "/";
+        $languages = array("fr", "en", "de");
+
+        foreach ($languages as $language) {
+            $transverse = $this->generateNodeTransverseGlobal($references, $language);
+            $manager->persist($transverse);
+        }
+
+        foreach ($languages as $language) {
+            $node = $this->generateNodeGlobal($references, $language, $routePattern);
+            $manager->persist($node);
+        }
+
+        $manager->flush();
+    }
+
+    /**
+     * Get the order of this fixture
+     *
+     * @return integer
+     */
+    public function getOrder()
+    {
+        return 560;
+    }
+
+    /**
+     * @param array $references
+     * @param string $language
+     * @param string $routePattern
+     *
+     * @return Node
+     */
+    protected function generateNodeGlobal($references, $language, $routePattern)
+    {
+        $siteHome = $this->createBaseNode($references);
+        $siteHome->setLanguage($language);
+        $siteHome->setNodeId(NodeInterface::ROOT_NODE_ID);
+        $siteHome->setName('Homepage');
+        $siteHome->setCreatedBy('fake_admin');
+        $siteHome->setParentId('-');
+        $siteHome->setOrder(0);
+        $siteHome->setRoutePattern($routePattern);
+        $siteHome->setInFooter(false);
+        $siteHome->setInMenu(true);
+        $siteHome->setSitemapChangefreq('hourly');
+        $siteHome->setSitemapPriority('0.8');
+
+        return $siteHome;
+    }
+
+    /**
+     * @param array $references
+     * @return Node
+     */
+    protected function createBaseNode($references)
+    {
+        $node = new Node();
+        $node->setMaxAge(1000);
+        $node->setNodeType(NodeInterface::TYPE_DEFAULT);
+        $node->setSiteId($references["siteId"]);
+        $node->setPath('-');
+        $node->setVersion("1");
+        $node->setStatus($references["status-published"]);
+        if ('status-published' == $references["status-published"]) {
```

Comment:
I think there's a problem here: $references["status-published"] is an object, you can't check it versus a string